src/Components/Calculator.jsx

```diff
@@ -1,45 +1,112 @@
-import React from 'react';
-import { useState } from 'react';
+import React, { useState } from "react";
 
 const Calculator = () => {
-    const Calculator = () => {
-        const [display, setDisplay] = useState('0');
-        const [firstOperand, setFirstOperand] = useState(null);
-        const [operator, setOperator] = useState(null);
-        const [waitingForSecondOperand, setWaitingForSecondOperand] = useState(false);
-    }
+  const [data, setData] = useState("");
 
+  const buttons = [];
 
+  [9, 8, 7, 6, 5, 4, 3, 2, 1, 0, ".", "%"].forEach((element) => {
```

Comment:
In javascript if there is only one parameter no need of putting it into parantheses